src/platform/Tizen/WiFiManager.cpp

```diff
@@ -15,14 +15,24 @@
  *    limitations under the License.
  */
 
-#include <platform/CHIPDeviceLayer.h>
+#include <platform/CHIPDeviceLayer.h> // IWYU pragma: keep
```

Comment:
The "WiFiManager.cpp" has to import corresponding header file as the first include, which is "WiFiManager.h" here. This way enforces that the includes in the "WiFiManager.h" are correct - compilation will fail otherwise.